mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -102,6 +103,27 @@ object VizReads extends ADAMCommandCompanion {
     }
     freqBuffer.toList
   }
+
+  def quit() {
+    val thread = new Thread {
```

Comment:
If you don't do it in a thread, the server will stop (throws an error about sleep being interrupted), but the VizReads process will keep running and won't hand back the terminal.